### PR TITLE
[stable/labelsmanager-controller]add labels manager controller

### PR DESCRIPTION
**Pull request by nyambati**
## Checklist

- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)
- [x] Github actions are passing

### CODE REVIEW

File: stable/labelsmanager-controller/values.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  # controller.metricAddr -- Address to serve prometheus metrics on. "0" is disabled.
+  metricAddr: "0"
+
+  # controller.overwrite -- Overwrite any similar label on the pod.
+  overwrite: false
+
+image:
+  repository: thomasnyambati/labelsmanager-controller
+  pullPolicy: IfNotPresent
+  tag: ""
```
Comment (max-rocket-internet) [inline stable/labelsmanager-controller/values.yaml:21]: Can you create a tag and put it here?

File: stable/labelsmanager-controller/values.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+podSecurityContext: {}
+
+securityContext: {}
+
+extraLabels: {}
+
+service:
+  type: ClusterIP
+  port: 80
+
```
Comment (max-rocket-internet) [inline stable/labelsmanager-controller/values.yaml:39]: Not used:

```suggestion
```

File: stable/labelsmanager-controller/values.yaml (hunk 1 above)
Comment (nyambati) [reply stable/labelsmanager-controller/values.yaml:21]: I am using the appVersion as default value, we can add this incase we use AWS registry

Comment (max-rocket-internet) [reply stable/labelsmanager-controller/values.yaml:21]: Ah I see. Nice. Just leave it like this then. I see it's the default behaviour in Helm now.

**Review by max-rocket-internet — APPROVED** [review]

**Pull request merged**